app/views/user_blocks/_navigation.html.erb

```diff
@@ -34,11 +34,18 @@
                   :class => ["nav-link", { :active => action_name == "blocks_by" }] %>
     </li>
   <% end %>
-  <% if @user_block %>
+  <% if @user_block&.persisted? %>
     <li class="nav-item">
       <%= link_to t(".block", :id => @user_block.id),
                   user_block_path(@user_block),
                   :class => "nav-link active" %>
     </li>
   <% end %>
+  <% if @user_block&.new_record? %>
```

Comment:
This could just be `elsif` instead of starting a new if?

Reply:
changed to `elsif`